jcoin.datacommons.io/portal/gitops.json

```diff
@@ -144,159 +144,192 @@
   "featureFlags": {
     "explorer": true
   },
-  "dataExplorerConfig": {
-    "charts": {
-      "project_id": {
-        "chartType": "count",
-        "title": "Projects"
+  "explorerConfig": [
+    {
+      "tabTitle": "Data",
+      "charts": {
+        "project_id": {
+          "chartType": "count",
+          "title": "Projects"
+        },
+        "gender": {
+          "chartType": "pie",
+          "title": "Gender"
+        },
+        "race": {
+          "chartType": "pie",
+          "title": "Race"
+        },
+        "center": {
+          "chartType": "bar",
+          "title": "center"
+        }
       },
-      "gender": {
-        "chartType": "pie",
-        "title": "Gender"
+      "filters": {
+        "tabs": [
+          {
+            "title": "Participant",
+            "fields": [
+              "project_id",
+              "gender",
+              "race",
+              "jewish",
+              "hispanic",
+              "drug_used",
+              "ever_transferred",
+              "pregnancy_status",
+              "protocol",
+              "weight",
+              "weight_percentage",
+              "bmi",
```

Comment:
Remove these also.

Reply:
which ones? weight, weight_percentage and bmi?

Reply:
Yes.